doc/src/manual/documentation.md

```diff
@@ -37,6 +37,8 @@ the documented object. Here is a basic example:
 "Tell whether there are too foo items in the array."
 foo(xs::Array) = ...
 ```
+!!! note "Reminder"
+    Having any lines between the docstring and the object being documented (function, macro, type or instance), while human readable, are not allowed and may break automated documentation tools.
```

Comment:
Disliking the current wording:

* `(function, macro, type or instance)`: seems redundant

* `while human readable`: subjective and unnecessary

* `may break automated documentation tools`: seems to weak/imprecise

I'm too tired to make a completely thoughtful suggestion, however maybe something like this would be an improvement?

```suggestion
    Any empty lines between the docstring and the object being documented detach the former from the latter, making the docstring useless.
```